packages/shared/src/components/post/collection/CollectionPostContent.tsx

```diff
@@ -139,6 +140,7 @@ export const CollectionPostContent = ({
             >
               {post.title}
             </h1>
+            <TagLinks tags={post.tags || []} />
```

Comment:
Should this be under feature flag?

Reply:
These are not for the experiment, just something we never added on the modal but should be there always regardless :)